tests/installation/select_patterns_and_packages.pm

```diff
@@ -60,6 +60,11 @@ sub gotopatterns {
         send_key 'alt-s';
     }
     else {
+        # See poo#12322. Prevent tab presses before overview is fully loaded
+        if (!check_screen('installation-settings-overview-loaded')) {
```

Comment:
Installation screen will be already there, but it may be in progress of loading. Here we additionally use "install" button which becomes active only when overview is loaded. send_key_until_needlematch still may succeed if there are less than 20 steps to reach software section. So I believe we should try to execute it further. Same story if needle doesn't much anymore, as it's the fix for the case which happens rarely on slow machines.